tests/rt.conf

```diff
@@ -1,275 +1,198 @@
-RUN     | fv3_ccpp_thompson_no_aero_debug                                                                                         | standard    |                | fv3         |
-RUN     | fv3_ccpp_rrfs_v1beta_debug                                                                                              | standard    |                | fv3         |
-
-COMPILE | SUITES=HAFS_v0_hwrf_thompson,HAFS_v0_hwrf DEBUG=Y                                                                       | standard    |                | fv3         |
-RUN     | fv3_ccpp_HAFS_v0_hwrf_thompson_debug                                                                                    | standard    |                | fv3         |
-#RUN     | fv3_ccpp_HAFS_v0_hwrf_debug                                                                                             | standard    |                | fv3         |
-RUN     | fv3_ccpp_esg_HAFS_v0_hwrf_thompson_debug                                                                                | standard    |                | fv3         |
-
-#######################################################################################################################################################################################
-# CPLD tests
-#######################################################################################################################################################################################
-
-COMPILE | SUITES=FV3_GFS_2017_coupled,FV3_GFS_2017_satmedmf_coupled,FV3_GFS_v15p2_coupled S2S=Y                        | standard | hera.intel orion.intel wcoss_dell_p3 | fv3 |
-#COMPILE | SUITES=FV3_GFS_2017_coupled,FV3_GFS_2017_satmedmf_coupled,FV3_GFS_v15p2_coupled S2S=Y                        | standard | cheyenne.intel                       | fv3 |
-RUN     | cpld_control                                                                                                 | standard | hera.intel orion.intel wcoss_dell_p3 | fv3 |
+COMPILE | DEBUG=Y                                                                                                                 | standard | + wcoss_dell_p3                         | fv3 |
+COMPILE | DEBUG=Y                                                                                                                 | standard | + wcoss_cray                            | fv3 |
+COMPILE | DEBUG=Y                                                                                                                 | standard | + hera.intel                            | fv3 |
+COMPILE | SUITES=FV3_GFS_v15p2,FV3_GFS_v16beta,FV3_GFS_v15p2_RRTMGP,FV3_GFS_v16beta_RRTMGP DEBUG=Y                                | standard | + orion.intel                           | fv3 |
+COMPILE | DEBUG=Y                                                                                                                 | standard | + cheyenne.intel                        | fv3 |
```

Comment:
May I ask why the suites are specified on other platforms, but not on cheyenne? Are there other suite files compiled on cheyenne besides those listed on the line above?

Reply:
The suites are only specified on orion.intel, but not on any of the other platforms (see lines 106-110). The reason is that compiling all suites on orion led to crash of the Intel compiler (some internal buffers overflowing or something like that).

Reply:
Do we know which suite files may cause issues on orion?

Reply:
I don't think a particular suite causes an issue, it is the fact that there are "too many" of them and the compiler optimization algorithm can't handle it. Why only on orion and not on any of the other platforms, I don't know. But this happens, for example due to some internal buffer overflow.

Reply:
I actually tried to compile the code on Orion with just "DEBUG=Y" and it worked. Compilation didn't crash. Maybe they updated compiler in the meantime or changed some other system configuration that was causing the crash before. I'm running a short test with this block in rt.conf

```
COMPILE | DEBUG=Y                            | standard | - gaea.intel jet.intel                  | fv3 |
RUN     | fv3_ccpp_gfs_v15p2_debug           | standard | + hera.intel orion.intel cheyenne.intel | fv3 |
RUN     | fv3_ccpp_gfs_v16beta_debug         | standard | - gaea.intel jet.intel                  | fv3 |
RUN     | fv3_ccpp_gfs_v15p2_RRTMGP_debug    | standard | + hera.intel orion.intel cheyenne.intel | fv3 |
RUN     | fv3_ccpp_gfs_v16beta_RRTMGP_debug  | standard | - gaea.intel jet.intel                  | fv3 |
```
If it passes we can make the change in one of the future commits.

Reply:
These 4 tests passed on Orion.